backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4115,4 +4115,18 @@ databaseChangeLog:
               FROM migration_user
       rollback:
         truncateTable:
-          tableName: feature_flag
\ No newline at end of file
+          tableName: feature_flag
+  - changeSet:
+      id: add-index-patient_link-created_at
+      author: zedd@skylight.digital
+      changes:
+        - createIndex:
+            tableName: patient_link
+            indexName: ix__patient_link__created_at
+            columns:
+              - column:
+                  name: created_at
+                  descending:  true
+      rollback:
+        - dropIndex:
+            indexName: ix__patient_link__created_at
```

Comment:
Adding an index to this will make the [`findMostRecentByTestOrderIdIn`](https://github.com/CDCgov/prime-simplereport/pull/4293/files#r975407868) query a lot faster since it orders by `created_at`